rpc/client_test.go

```diff
@@ -91,7 +91,7 @@ func TestChainID(t *testing.T) {
 		ChainID string
 	}
 	testSet := map[string][]testSetType{
-		"devnet":  {{ChainID: "DEVNET"}},
+		"devnet":  {{ChainID: "SN_GOERLI"}},
```

Comment:
I would think it is a bug on starknet-devnet. I  have open a ticket https://github.com/Shard-Labs/starknet-devnet/issues/146